test/tree/OperatorStateRetrieverUnit.tree

```diff
@@ -0,0 +1,35 @@
+.
+└── OperatorStateRetriever tree (*** denotes that integration tests are needed to validate path)
+    ├── when getOperatorState(IRegistryCoordinator,bytes32,uint32) is called
+    │   ├── when the given operator has never registered with the given registryCoordinator (their quorumBitmapHistory has length 0)
```

Comment:
Perhaps `_operatorBitmapHistory` is more explicit and accurate here instead of `quorumBitmapHistory`